codespan-reporting/tests/snapshots/term__multiline_omit__rich_no_color.snap

```diff
@@ -0,0 +1,35 @@
+---
+source: codespan-reporting/tests/term.rs
+expression: test_data.emit_no_color(&config)
+---
+error[empty_if]: empty elseif block
+   ┌─ empty_if_comments.lua:1:1
+   │    
+ 1 │ ╭   elseif 3 then
+ 2 │ │   
+ 3 │ │ ╭ 
+ 4 │ │ │ 
+   · │ │
```

Comment:
Yeah I like this rendering better - ie. continuing the lines over the source break. Good stuff 👍

Reply:
Noticed that you already thought ahead with `render_snippet_break`. I just had to put in the multilables on that line so it renders like that. 😄